README.md

```diff
@@ -28,6 +28,9 @@ To install the C++ API from precompiled binaries you'll need a few things;
   release](https://github.com/bytecodealliance/wasmtime/releases) (an artifact
   with "c-api" in the name).
 
+* Next you'll want to copy [`include/gsl`](https://github.com/microsoft/GSL/tree/main/include/gsl) into
+  the directory `gsl` when you want.
```

Comment:
I think this may be stale at this point?

Reply:
Removed